### PR TITLE
WIP: Add a recipe for PyQt5

### PR DESCRIPTION
**Pull request by ccordoba12**
Once Qt5 is in the official channels, this one should work without problems here :-)

Pinging @msarahan and @mingwandroid about it.

### CODE REVIEW

**Comment by ccordoba12** [comment]
Also pinging @astrofrog :-)

**Comment by astrofrog** [comment]
@ccordoba12 - thanks! Note that as mentioned in https://github.com/conda/conda-recipes/issues/385 it might be necessary to deal with the ``QT_QPA_PLATFORM_PLUGIN_PATH`` environment variable?

File: recipes/pyqt5/bld.bat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+::CALL "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x64
```
Comment (patricksnape) [inline recipes/pyqt5/bld.bat]: Is there a reason for this commented call to the Windows SDK?

File: recipes/pyqt5/bld.bat [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+%PYTHON% configure.py ^
+        --spec=win32-msvc2010 ^
```
Comment (patricksnape) [inline recipes/pyqt5/bld.bat]: This is only true for Python 3.4?

Comment (ccordoba12) [inline recipes/pyqt5/bld.bat]: Sorry, we haven't had time to work on the Windows part :-) It'll come later this week.

**Comment by ccordoba12** [comment]
@astrofrog, there's no need of that if you update `qt` and `pyqt` to their latest versions. We recompile them last week to make them work side by side with `qt5` and `pyqt5`.

----

You can also try our new `qt5` and `pyqt5` packages (that support Qt 5.6 and WebEngine) for OS X with

    conda install -c spyder-ide pyqt5

We're working on compiling Qt5 for the other OSes right now :-)

**Comment by ccordoba12** [comment]
Closing this one as there's now a pyqt feedstock. I'll move this recipe there after we compile Qt5 in the qt feedstock ;-)

**Pull request closed without merging**